implementations/micrometer-registry-dynatrace/src/main/java/io/micrometer/dynatrace/v2/DynatraceExporterV2.java

```diff
@@ -139,9 +136,22 @@ public void export(List<Meter> meters) {
         // Lines that are too long to be ingested into Dynatrace, as well as lines that
         // contain NaN or Inf values are not returned from "toMetricLines", and are
         // therefore dropped.
```

Comment:
This comment should probably also be moved to where `toMetricLines` is being called.

Reply:
Done. Moved the comment closer to ` toMetricLines`